src/server/autocomplete.d

```diff
@@ -73,6 +73,15 @@ public AutocompleteResponse getDoc(const AutocompleteRequest request,
 	{
 		Appender!(char[]) app;
 
+		bool isDitto(string s)
+		{
+			import std.string: toLower;
+			if (s.length > 6)
+				return false;
+			else
+				return s.toLower == "ditto";
```

Comment:
I don't know why the new GH interface doesn't collapse your comment but agreed and done.